tools/perf-automation/Azure.Sdk.Tools.PerfAutomation/Java.cs

```diff
@@ -0,0 +1,103 @@
+﻿using Azure.Sdk.Tools.PerfAutomation.Models;
+using System.Collections.Generic;
+using System.IO;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using System.Xml;
+
+namespace Azure.Sdk.Tools.PerfAutomation
+{
+    public class Java : LanguageBase
+    {
+        protected override Language Language => Language.Java;
+
+        private string perfCoreProjectFile => Path.Combine(WorkingDirectory, "common", "perf-test-core", "pom.xml");
+
+        public override async Task<(string output, string error, string context)> SetupAsync(
+            string project, string languageVersion, IDictionary<string, string> packageVersions)
+        {
+            var projectFile = Path.Combine(WorkingDirectory, project, "pom.xml");
+
+            UpdatePackageVersions(perfCoreProjectFile, packageVersions);
+            UpdatePackageVersions(projectFile, packageVersions);
+
+            var result = await Util.RunAsync("mvn", $"clean package -T1C -am -Denforcer.skip=true -DskipTests=true -Dmaven.javadoc.skip=true --pl {project}",
```

Comment:
Hard coding these commands into this program is going to make maintaining this tool very expensive longer term we should figure out a way to abstract this so we can manage the build steps inside the language repos.